src/github.ts

```diff
@@ -148,6 +153,7 @@ export class OctokitGitHubClient {
     path,
   }: GetTreeSHAForPathOptions): Promise<string | null> {
     const { owner, repo } = parseRepoURL(repoURL);
+    core.info(`GH API: repos.getContent ${owner}/${repo} ${commitSHA}`);
```

Comment:
nit: Want to make these debug, or do you want them on by default?

Reply:
Eh, at this point having it in seems useful enough.  Might clean up all the logging as part of making parallelism log well, though.